test/checkpoint/workers.c

```diff
@@ -59,8 +59,8 @@ create_table(WT_SESSION *session, COOKIE *cookie)
           "memory_page_max=64KB,log=(enabled=false),%s",
           kf, vf, lsm));
     else
-        testutil_check(
```

Comment:
Can you fix the comment above as we are turning off the logging for both timestamped and non-timestamped scenarios?

Reply:
thanks, fixed in a715739